bin/bundler-search

```diff
@@ -12,4 +12,5 @@
 #  2. Which gem names to search (defaults to all gems)
 
 pattern="$1"; shift
+# shellcheck disable=SC2046
```

Comment:
What is this ignoring?

I think it's worse to have this comment than to have shellcheck fail, because this comment is totally opaque.

When/if we run shellcheck against this again, I think we can make a judgment call.